include/mme-app/contextManager/dataBlocks.h

```diff
@@ -388,7 +402,10 @@ namespace mme
 			
 			// DataName
 			Tai tai_m;
-			
+		
```

Comment:
This needs to be added via ctxtManagerAppModel.json. Have a look at [documentation](https://github.com/omec-project/Nucleus/blob/master/Documentation/Documentation.md#code-generation-for-state-machine-and-context-manager) on how to do it. In case of any queries, kindly mail me.